src/main/java/edu/hm/hafner/analysis/parser/QtTranslationSaxParser.java

```diff
@@ -0,0 +1,211 @@
+package edu.hm.hafner.analysis.parser;
+
+import java.util.ArrayDeque;
+import java.util.Deque;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.commons.lang3.StringUtils;
+import org.xml.sax.Attributes;
+import org.xml.sax.Locator;
+import org.xml.sax.helpers.DefaultHandler;
+
+import edu.hm.hafner.analysis.IssueBuilder;
+import edu.hm.hafner.analysis.ParsingException;
+import edu.hm.hafner.analysis.Report;
+import edu.hm.hafner.analysis.Severity;
+
+/**
+ * Handles the parsing of a translation file from Qt.
+ */
+public class QtTranslationSaxParser extends DefaultHandler {
```

Comment:
Please make this a static nested class of `QtTranslationParser`.

Reply:
Okay, I did this on this way as I uses as base `LintParser` & `JSLintXmlSaxParser`